x-pack/solutions/observability/plugins/uptime/public/legacy_uptime/lib/helper/observability_integration/get_apm_href.ts

```diff
@@ -19,7 +19,9 @@ export const getLegacyApmHref = (
   if (serviceName) {
     return addBasePath(
       basePath,
-      `/app/apm/services/${serviceName}/overview/?rangeFrom=${dateRangeStart}&rangeTo=${dateRangeEnd}`
+      `/app/apm/services/${encodeURIComponent(
```

Comment:
This is probably not needed as it's legacy_uptime but changed that and the test anyway